user_docs/en/changes.t2t

```diff
@@ -31,8 +31,16 @@ What's New in NVDA
 == Changes for Developers ==
 - Compiling NVDA dependencies with Visual Studio 2022 (17.0) is now supported. For development and release builds, Visual Studio 2019 is still used. (#13033)
 - When retrieving the count of selected children via accSelection, the case where a negative child ID or an IDispatch is returned by IAccessible::get_accSelection is now handled properly. (#13276)
+-
+
+=== Deprecations ===
+These are proposed API breaking changes.
+Until a specified date, both the current and new API will work.

```

Comment:
Not sure about this.
- We don't always specify a date, I.E. we may be undecided if / when it will be removed, but we'd recommend against introducing usages of it.
- There may not be a direct replacement.

```suggestion
The deprecated part of the API will continue to be available until the specified release.
If no release is specified, the plan for removal has not been determined.
Note, the roadmap for removals is 'best effort' and may be subject to change.
```

Reply:
Agreed, good catch on both of these. I'll follow this up with a PR to fix our template for this.